myapp/src/components/InputField.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This is a cutome input field.
+ * This all params are required.
+ *
+ * @param {string} title - use for input labaling
+ * @param {string} value - user given current value
+ * @param {string} type - input tag type
+ * @param {Function} [ e => setfield(e.target.value)] onChange - call back function
+ * @param {string} error - display an error if the user given data invalid
+ * @returns {Component}  - return a input  component
+ */
+
+function InputField({ title, value, type, onChange, error }) {
+  // when error occured change border style
+  const styleBorder = error !== "" ? "border-red-500" : "";
+
+  const style = {
+    lable: "block pb-2 text-xl text-purple-700 focus:border-violet-50",
```

Comment:
```suggestion
    label: "block pb-2 text-xl text-purple-700 focus:border-violet-50",
```
There could be even the slightest spelliing errors.

Reply:
Changed it.